chapter_recurrent-neural-networks/language-models-and-dataset.md

```diff
@@ -124,7 +124,7 @@ bigram_vocab = d2l.Vocab(bigram_tokens)
 print(bigram_vocab.token_freqs[:10])
 ```
 
-Two things are notable. Out of the 10 most frequent word pairs, 9 are composed of stop words and only one is relevant to the actual book---"the time". Furthermore, let us see whether the trigram frequency behaves in the same manner.
+There is one thing notable here. Out of the 10 most frequent word pairs, 9 are composed of stop words and only one is relevant to the actual book---"the time". Furthermore, let us see whether the trigram frequency behaves in the same manner.
```

Comment:
one notable thing?

Reply:
I think so. It is just mentioned in the subsequent sentence:
> Out of the 10 most frequent word pairs, 9 are composed of stop words and only one is relevant to the actual book---"the time".

Reply:
I mean, one thing notable -> one notable thing

Reply:
or something notable